### ISSUE TITLE
Feature request: Add optional config setting to remove watched items from watchlists

### ISSUE THREAD

**Issue by RileyXX** [Taxel/PlexTraktSync#1494]
### Feature description

Add optional config setting to remove watched items from watchlists.

### Use case

For users who want to keep their watchlists organized with only content they have not yet view.

### Plan to implement

When I get some time I might consider making a PR request for this issue. But it will probably be easier for someone with more experience in this repo to implement this.

**Comment by glensc** [comment]
Should the setting be global? or trakt and plex should have own option?

**Comment by simonc56** [comment]
As far as I know, Trakt automatically removes watched items from watchlist.

From [trakt.tv](https://blog.trakt.tv/manage-your-watchlist-personal-lists-9e4fdc521e3d) :
>Once you watch an item from your watchlist, it automatically gets removed.

**Comment by glensc** [comment]
That doesn't really give answer.

As you can still put items to trakt playlist if the item is watched. trakt only removes when scrobble ends or you add watch date. so you could still use plextraktsync to sync over. I suppose their use case would be that you want to rewatch.

so the flag would make sense for sync.plex_to_trakt as well

also, there's trakt lists,  this option would only be effect on watchlist, not trakt lists? if trakt/plex lists it probably should be configurable per list.

**Comment by glensc** [comment]
I mean pts can add item back to trakt watchlist if it's present in plex watchlist. in response to your statement:
- https://github.com/Taxel/PlexTraktSync/issues/1494#issuecomment-1894120548

**Comment by simonc56** [comment]
> I mean pts can add item back to trakt watchlist if it's present in plex watchlist. in response to your statement:
> 
>     * [Feature request: Add optional config setting to remove watched items from watchlists #1494 (comment)](https://github.com/Taxel/PlexTraktSync/issues/1494#issuecomment-1894120548)

No, pts will not add it back because it will be already removed from Plex watchlist with the setting remove_watched_from_watchlist. If you disagree, can you show detailed usecase step by step.

**Comment by glensc** [comment]
you are talking about the future. but I'm speaking of current behavior and when remove_watched_from_watchlist = false.

pointless discussion. all I wanted to say that your statement is not answering the question should it be global or plex only option. because you can add watched item in trakt watchlist.

**Comment by simonc56** [comment]
Usecases are important. No need to code things for impossible usecase.

**Comment by glensc** [comment]
it's simple usecase: keep trakt watchlist synced with plex watchlist, like it's now. whether it's practical, is other question. but since the request here is to make it configurable, it's practical for someone.

**Comment by WladyX** [comment]
After I watch a movie on plex I go to remove it from watchlist + remove it from watchlist from trakt.
With this functionality trakt would remove it automatically after i watch it from both lists, so nothing for me to do.
2 manual actions vs no manual action.
+1 for this feature :)

**Comment by glensc** [comment]
and I explained that you can use pts to sync the status back.

**Comment by Bart1909** [comment]
Any news on this?

For me the usecase would be: 
Using a trakt liked playlist. So if I watch an episode in plex it will not being removed from trakt playlist because I do not own this playlist.
So it would be very nice, if this episode would then be removed from the plex playlist

**Comment by glensc** [comment]
watched lists are always yours, what you probably have in mind is liked lists, their behaviour is somewhat different anyway. this issue is about watch list syncing.

**Comment by Bart1909** [comment]
Okay, should I open an additional feature request for this?

**Comment by glensc** [comment]
@Bart1909 if you want the feature then yes, you should describe what you want.
